### DISCUSSION TITLE
Expanded Drag & Drop Features in v2.18.0

### DISCUSSION THREAD

**Discussion by okaryo** [okaryo/TabTabTab#169] [Announcements]
Hi! 🖐️

I'm excited to announce that the Drag & Drop functionality has been expanded in [v2.18.0](https://github.com/okaryo/TabTabTab/releases/tag/2.18.0). While previously you could only rearrange tabs directly under a window, with this new release, you can also reorder pinned tabs and tabs within a TabGroup. This means you can now move tabs from one TabGroup to another or drag them out of a TabGroup or into the pinned section, all within TabTabTab.

If you find any bugs or face any issues while using this new feature, please don't hesitate to let us know. Your feedback helps us make the tool even better!

Looking ahead, here's what I'm planning for future development. If there are other improvements or features you'd like to see, please share your thoughts:

* Refining the existing Drag & Drop functionality. 🛠️
* Enabling moving tabs to a different window using Drag & Drop. ↗️

Thank you for being a part of our community and for helping us improve! 💬🚀

**Comment by FreePDF64** [comment]
Thanks for the further development of TabTabTab!

I am very excited about the implementation, but still have to wait for the Chrome Web Store synchronization.

I also hope for a customized description of the AddOn where this feature is highlighted. There are few tab managers which also manage tab groups. Do they exist at all?

**Comment by okaryo** [comment]
The store review might take a little more time.

I forgot to update the description. I'll make sure to update it in the next release. Thanks!
